pkg/controller/bypass/bypass_controller.go

```diff
@@ -124,6 +148,85 @@ func StartByPassController(client kubernetes.Interface) error {
 	return nil
 }
 
+func bypassKmeshControl(ns string) error {
+	execFunc := func(netns.NetNS) error {
+		/*
+		 * Attempt to connect to a special IP address. The
+		 * connection triggers the cgroup/connect4 ebpf
+		 * program and records the netns cookie information
+		 * of the current connection. The cookie can be used
+		 * to determine whether the netns is managed by Kmesh.
+		 * 0.0.0.1:931(0x3a3) is "cipher key" for cgroup/connect4
+		 * ebpf program.
+		 */
+		simip := net.ParseIP("0.0.0.1")
+		sockfd, err := syscall.Socket(syscall.AF_INET, syscall.SOCK_STREAM, 0)
+		if err != nil {
+			return err
+		}
+		if err = syscall.SetNonblock(sockfd, true); err != nil {
+			return err
+		}
+		err = syscall.Connect(sockfd, &syscall.SockaddrInet4{
+			Port: 931,
+			Addr: [4]byte(simip.To4()),
+		})
+		if err == nil {
+			return err
+		}
+		errno, ok := err.(syscall.Errno)
+		if ok && errno == 115 { // -EINPROGRESS, Operation now in progress
+			return nil
+		}
+		return err
+	}
+
+	if err := netns.WithNetNSPath(ns, execFunc); err != nil {
+		err = fmt.Errorf("enter ns path :%v, run execFunc failed: %v", ns, err)
+		return err
+	}
+	return nil
+}
+
+func enableKmeshControl(ns string) error {
+	execFunc := func(netns.NetNS) error {
+		/*
+		 * Attempt to connect to a special IP address. The
+		 * connection triggers the cgroup/connect4 ebpf
+		 * program and records the netns cookie information
+		 * of the current connection. The cookie can be used
+		 * to determine whether the netns is managed by Kmesh.
+		 * 0.0.0.1:932(0x3a4) is "cipher key" for cgroup/connect4
+		 * ebpf program.
+		 */
+		simip := net.ParseIP("0.0.0.1")
```

Comment:
execFunc considers whether it is possible to extract

Reply:
> execFunc considers whether it is possible to extract

done